nubis/bin/ldap2consul.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
+
+import os
+import sys
+import ConfigParser
+import ldap
+import consul
+
+_config = {}
+
+def load_config(config_file):
+    global _config
+    if not os.path.exists(config_file):
+        print "ERROR: Config file %s not found." % config_file
+        sys.exit(-1)
+    config = ConfigParser.RawConfigParser()
+    config.read(config_file)
+    if 'ldap' not in config.sections():
+        print "ERROR: [ldap] section not found in %s." % config_file
+        sys.exit(-1)
+    _config = config
+
```

Comment:
Apart from making sure the config block exists, no configuration validation is performed.